registry/remote/repository_test.go

```diff
@@ -2455,6 +2452,16 @@ func Test_generateBlobDescriptorWithVariousDockerContentDigestHeaders(t *testing
 	}
 }
 
+func TestManifestStoreInterface(t *testing.T) {
```

Comment:
same here

Reply:
I think we can test `interfaces.ReferenceParser` here.

Reply:
Kept `ReferenceParser` test and removed the rest.